include/constants/metatile_behaviors.h

```diff
@@ -242,4 +242,96 @@
 #define MB_UNUSED_EE 0xEE
 #define MB_UNUSED_EF 0xEF
 
+// Raw metatile ID's.
+#define METATILE_ID_UNK_206 0x206
```

Comment:
metatile ids do not belong in the metatile behaviors header file.  At minimum, this should be its own file, but I think individual C files should be using their own `#define` values for this, rather than creating a header file.